config.py

```diff
@@ -76,10 +76,17 @@ class Config():
 
     LOG_TO_STDOUT = os.environ.get('LOG_TO_STDOUT')
 
+    MAX_CONTENT_LENGTH = 1024 * 1024
```

Comment:
Ah! So here is a config that ensures you don't upload something massive 👍

If you break this rule, what kind of error do you get? It's good to test this as you want to display that in the frontend

Reply:
Good point! I will do some tests with this.

Reply:
Additionally, by using automatic resizing with Pillow, we can prevent this from happening at all. See issue #22